comptests/motr/test_motr_S3bench_DI_tests.py

```diff
@@ -0,0 +1,23 @@
+     return list_s3bench_DI_comb
+
+def main():
+    acsKey    = args.a
+    secretKey = args.s
+    endPoint  = args.e
+    for test in s3bench_DI_combinations():
+        command = "s3bench -region us-east-1 -accessKey {} -accessSecret {} -bucket test1 -endpoint {} -numClients {} -numSamples {} -objectSize {}".format(acsKey, secretKey, endPoint, test['noOfClients'], test['noOfSamples'], test['objectSize'])
```

Comment:
We are just running s3bench here in this loop, what else are we doing to test the DI feature here?